src/utils/optionsHaveIgnoredPseudoClass.js

```diff
@@ -0,0 +1,17 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
```

Comment:
None of the existing `option*` utils have tests. 

Is it redundant to add them?